### PR TITLE
Toolview

### PR DESCRIPTION
**Pull request by Markvanderuit**
Simple layout for toolview and its content.

Also adds a TweakingHelper class, which contains 8 statics to control color scheme app-wide.

### CODE REVIEW

File: src/main/java/gui/headerarea/ToolView.java [hunk 1]
```diff
@@ -15,6 +17,12 @@
 
     private List<ToolButton> buttonList;
 
+    private String style = "-fx-background-color: "
```
Comment (bart-de-jonge) [inline src/main/java/gui/headerarea/ToolView.java:20]: why do you hardcode the styles here? I though you were going to use css files?

**Comment by bart-de-jonge** [comment]
One small comment, other than that great work!

File: src/main/java/gui/headerarea/ToolView.java (hunk 1 above)
Comment (Markvanderuit) [inline src/main/java/gui/headerarea/ToolView.java:20]: Can't couple that well between java and css, sadly. This means I'll lose a lot of the tweakability changing variables/properties gives me. :(

**Comment by mennoo1996** [comment]
Good work!

**Pull request merged**